### ISSUE TITLE
Quick fix verse ref invalidations

### ISSUE THREAD

**Issue by ryderwishart** [genesis-ai-dev/codex-editor#25]
Refs [#10](https://github.com/genesis-ai-dev/codex-editor/issues/10)

- [x] If a verse is duplicated, we should offer to remove the vref.
- [x] If a verse is missing, we should offer to add it.

I _think_ this allows the user to 'quick fix' all the problems we detect, even if it may take multiple steps. There are probably some cases I'm not thinking of.

**Comment by dadukhankevin** [comment]
@ryderwishart Thanks, I was also wondering where the file with all the verse references is, is it on ebible?

**Comment by BenjaminScholtens** [comment]
@dadukhankevin Yes, the vrefs in the codex project are from the ebible https://github.com/BibleNLP/ebible/blob/main/metadata/vref.txt

We simply took that file and its located at servers/versedata.txt in the codex repo now

**Comment by dadukhankevin** [comment]
Alright, it should be complete, for now, if there is a duplicate verse I don't have it delete the actual text associated with the duplicate because I'm assuming that could lead to someone accidentally deleting good text and not realizing it. I can change that though if it's better to delete the whole thing. Also, now all of it works in inline verse references which it wasn't.
